package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "seed-db": "node ./seeder/seedingScript.js",
-    "reset-db": "mysql -u root <schema.sql",
+    "reset-db": "mysql -u root -p <schema.sql",
```

Comment:
this would only work if root has no password.
recommend change this to

"reset-db": "mysql -h `echo $DB_HOST` -u `echo $DB_USER` -p`echo $DB_PASSWORD` <schema.sql",